src/TSnePlotView.js

```diff
@@ -17,8 +17,8 @@ class TSnePlotView extends React.Component {
     super(props)
     this.state = {
       geneExpressionData: {
-        max: null,
-        min: null,
+        max: 0,
```

Comment:
The null is there for a reason. Check the JSON response from the backend if a gene is not expressed in an experiment.